tests/fakeapi/hello/__init__.py

```diff
@@ -343,40 +343,101 @@ async def test_formdata_file_upload(file):
 
 async def test_formdata_multiple_file_upload(file):
     """In Swagger, form parameters and files are passed separately"""
```

Comment:
Just trying to understand, this comment seems to suggest that the test case is for Swagger only, but maybe it's actually indicating that in both Swagger (openapi 2) and Open API 3, form parameters and files are passed separately?